tests/unit/module_utils/test_dataset_utils.py

```diff
@@ -700,3 +708,147 @@ def test__write_jcl_to_data_set_fail():
 
     assert e.value.message == "Failed to copy JCL content to data set"
     assert e.value.executions == expected_executions
+
+def test_data_set_utils_submit_jcl():
```

Comment:
Dont think the test methods need data_set_utils in them, the files already named accordingly